content/applications/hr/referrals.rst

```diff
@@ -0,0 +1,703 @@
+in the lower-left corner of the screen. A :guilabel:`Create a post` window loads, with the job
+position link populated. Enter any other information to the post in the field.
+
+To change the audience that will see the post, click the :guilabel:`🌎 (globe) Anyone` drop-down,
+located directly beneath the account name. Click on the desired selection, then click the
+:guilabel:`Save` button on the far right side of the screen, beneath the last listed option.
+
+:guilabel:`Everyone` allows anyone on LinkedIn to see the post. :guilabel:`Connections only` will
+only share the post with people who are connected with the user, and will be hidden from everyone
+else. :guilabel:`Group` shares the post with a specific group the user is a member of. The post can
+only be shared with one group; posting to multiple groups at once is not available. If
+:guilabel:`Group` is selected, a page loads with a list of all the groups the user is currently a
+member of. Click on the group to select it, then click the :guilabel:`Save` button in the lower
+right corner.
+
+Next, select who can comment on the post. To change this setting, click on the :guilabel:`🗩 (speech
+bubble) Anyone` button located beneath the body of the post. A page loads with the various options.
+Click on one of the three options to select it. The three options are :guilabel:`Anyone`,
+:guilabel:`Connections only`, and :guilabel:`No one`. The default option is :guilabel:`Anyone`,
+which allows anyone with a LinkedIn account to comment on the post. :guilabel:`Connections only`
+will only allow people who are connected with the user to comment on the post. :guilabel:`No one`
+turns off commenting, not allowing anyone to post. After making a selection, click the
+:guilabel:`Save` button in the far right, beneath the list of options.
+
+Finally, once the post is edited and the settings have been selected, click :guilabel:`Post` to
+share the job position.
+
+.. image:: referrals/linkedin.png
+   :align: center
+   :alt: A linkedIn post to share a job. The job position is highlighted as is the post button.
+
+Send as private message
+***********************
+
+Instead of making a public post, the post can be sent to a connection in a private message. To do
+so, click the :guilabel:` Send as private message` button in the lower left corner. A message window
+loads with the link to the job description in the bottom of the window. First, select the
+recipient(s) for the message. Type the name in the :guilabel:`Search for message recipients` field.
+As the name is typed in, any matching connections appear. Click on a connection to add them. Repeat
+this for all the recipients to send the message to. There is no limit on the number of connections
+that can be messaged. Next, enter any additional text in the message body. Click :guilabel:`Send` in
+the lower right corner to send the message. A `Message successfully sent!` screen loads. Two options
```

Comment:
I am torn because you don't click on it- so I am never sure for these situations I thought GUI's were for clickable items. I'll change to a GUI because I think you are right- it also looks better.